src/DonationForms/resources/app/utilities/getDefaultValuesFromSections.ts

```diff
@@ -13,9 +13,7 @@ export default function getDefaultValuesFromSections(sections: Section[]) {
     return reduceFields(
         sections,
         (values, field) => {
-            if (field.defaultValue !== null) {
-                values[field.name] = field.defaultValue;
-            }
+            values[field.name] = field.defaultValue ?? '';
```

Comment:
@pauloiankoski I had to remember why we did this which conflicts with what your trying to do.  Let me explain:

The default values hold the key / value pairs of all our field names and their default values.  When the fields get registered by RHF, it uses what we define here to register the field.  I think you already understand that part.  We also intentionally exclude the field key in this if there is no default value.

There are 2 tricky pieces to this:
1. We can't assume that every input accepts (or wants) a string for a default value
2. On multi-step forms - each step uses the cascading default values - so its important that default values without a value are omitted.

So I think what we had originally is what we want to keep and should try handling undefined default values on a per-template basis - since currently it seems to only be problematic for controller based fields like the date one.

Reply:
Replaced the `Controller` component we had there with a hidden input.